README.md

```diff
@@ -468,6 +468,9 @@ fastify.get('/responseDescription', {
   }
 }, () => {})
 ```
+
+Additionally, if you provide a `$ref` in your response schema but no description, if your `$ref` base schema includes a description, that will be used.
```

Comment:
I am not sure that's very clear. Do you think you should rephrase it?

Reply:
I can try, how's about something along the lines of:

```
Additionally, if you provide a `$ref` in your response schema but no description, the reference's description will be used as a fallback.
```